common/containers/Tabs/SignAndVerifyMessage/components/SignMessage/SignButton.tsx

```diff
@@ -25,7 +25,7 @@ export default class SignMessageButton extends React.Component<Props, {}> {
 
     try {
       const signedMessage: ISignedMessage = {
-        address: await wallet.getAddressString(),
+        address: wallet.getAddressString(),
```

Comment:
This function no longer needs to be async.

Reply:
It does due to `sig: await wallet.signMessage(message),` unless I am mistaken

Reply:
Well butter my biscuits, you're right. My bad.